src/register/register.service.ts

```diff
@@ -41,7 +41,7 @@ export class RegisterService {
       await this.mailService.sendMail(
         student.email,
         'AKTYWUJ KONTO NA PLATFORMIE HEADHUNTERS by MegaK',
-        `Aby dokończyć rejestrację na platformie HEADHUNTERS by MegaK, kliknij w ten link: http://localhost:3001/register/${user.id}/${user.registerToken}.`,
+        `Aby dokończyć rejestrację na platformie HEADHUNTERS by MegaK, kliknij w ten link: ${config.APP_DOMAIN}/register/${user.id}/${user.registerToken}.`,
```

Comment:
Nie na wszystkich serwerach pocztowych ten link jest klikalny we wiadomościach. Proponuję cos w tym stylu:"        `Aby dokończyć rejestrację na platformie HEADHUNTERS by MegaK, kliknij w ten link: <a href="${config.APP_DOMAIN}/register/${user.id}/${user.registerToken}">Kliknij mnie</a>`,"

Reply:
to samo w linijce 54